omod/src/main/webapp/pages/conditionlist/manageCondition.gsp

```diff
@@ -70,17 +71,18 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                         datePickerFormat : "yyyy-MM-dd",
                         label            : ui.message('coreapps.stopDate.label'),
                         useTime          : false,
-                        endDate          : new Date()
+                        endDate          : new Date(),
+                        clearButton      : true
                 ])}
             </li>
             <br/><br/>
             <div id="status" class="horizontal">
                 <p>
-                    <input type="radio" id="status-1" class="condition-status" value="${ui.message('coreapps.conditionui.active.label')}" name="status" ng-model="condition.status"  ng-change="showEndDate()"/>
+                    <input type="radio" id="status-1" class="condition-status" value="ACTIVE" name="status" ng-model="condition.status"  ng-change="showEndDate()"/>
                     <label for="status-1">${ui.message('coreapps.conditionui.active.label')}</label>
                 </p>
                 <p>
-                    <input type="radio" id="status-2" class="condition-status" value="${ui.message('coreapps.conditionui.inactive.label')}" name="status" ng-model="condition.status"  ng-change="showEndDate()"/>
+                    <input type="radio" id="status-2" class="condition-status" value="INACTIVE" name="status" ng-model="condition.status"  ng-change="showEndDate()"/>
                     <label for="status-2">${ui.message('coreapps.conditionui.inactive.label')}</label>
```

Comment:
In testing with languages other than English, I found things to fail when trying to set inactive/active status.  I found this was due to the above, where the value of the status field was varying based on locale, rather than representing the enum value.

Reply:
:)
Thanks!